internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -92,13 +168,33 @@ func checkExists(resourceName string) resource.TestCheckFunc {
 	}
 }
 
-func resourceConfigBasic(projectID, endpointID string) string {
+func resourceConfigBasic(projectID, endpointID, comment string) string {
 	return fmt.Sprintf(`
 	resource "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive" "test" {
 	  project_id				= %[1]q
 	  endpoint_id				= %[2]q
 	  provider_name				= "AWS"
-	  comment					= "Terraform Acceptance Test"
+	  comment					= %[3]q
+	}
+	`, projectID, endpointID, comment)
+}
+
+func asCustomerEndpointDNSName(endpointID string) string {
```

Comment:
```suggestion
func customerEndpointDNSName(endpointID string) string {
```

Reply:
or toCustomerEndpointDNSName or something?

Reply:
Renamed to `toCustomerEndpointDNSName`, as I'm using customerEndpointDNSName as the variable name